resources/desktop/plexmediaplayer.desktop

```diff
@@ -3,7 +3,7 @@ Version=1.0
 Name=Plex Media Player
 GenericName=Media Player
 Comment=View your media
-Exec=plexmediaplayer --fullscreen --tv
+Exec=plexmediaplayer
```

Comment:
Good fix, thanks! Curious this made it through code review last time. 

Tangentially, is `Actions=TV;DesktopF;DesktopW` correct as well?  Is the convention that that it is `{theAction}` in the `[Topic Action {theAction}]` headers?

Reply:
Yes, it is correct. Well it's just a class name for what Exec runs and Name displays.

There might also be some benefit to add `TVW` to Actions
```
[Desktop Action TVW]
Name=TV [Windowed]
Exec=plexmediaplayer --windowed --tv
```
since it is the only layout missing.
And then I guess renaming `TV` to `TVF` and adding the `[Fullscreen]` description to keep it tidy.

Reply:
`TVW` and `TVF` sound great. It's possible `TV` forcing `--fullscreen` was a carryover from the past when we used to enforce that TV was fullscreen by default. 

I can't see a downside restoring the last state by default or to a specific state using one of the exec blocks consistently across layouts.